Polyform-Small.md

```diff
@@ -46,7 +46,7 @@ You may have "fair use" rights for the software under the law. These terms do no
 
 ## Small Business
 
-Use of the software for the benefit of your company is use for a permitted purpose if your company has less than 100 total individuals working as employees and independent contractors, and less than 1,000,000 USD (2019) total revenue in the prior tax year.  Adjust this revenue threshold for inflation according to the United States Bureau of Labor Statistics' consumer price index for all urban consumers, U.S. city average, for all items, not seasonally adjusted, with 1982–1984=100 reference base.
+Use of the software for the benefit of your company is use for a permitted purpose if your company has less than 100 total individuals working as employees and independent contractors, and less than 1,000,000 USD total revenue in your prior fiscal or tax year, adjusted for inflation according to the United States Bureau of Labor Statistics' consumer price index for all urban consumers, U.S. city average, for all items, not seasonally adjusted, with 1982–1984=100 reference base.
```

Comment:
@heathermeeker I think we need to say that the stated amount is 2019 dollars in order to adjust according to CPI.

```suggestion
Use of the software for the benefit of your company is use for a permitted purpose if your company has less than 100 total individuals working as employees and independent contractors, and less than 1,000,000 USD (2019) total revenue in your prior fiscal or tax year, adjusted for inflation according to the United States Bureau of Labor Statistics' consumer price index for all urban consumers, U.S. city average, for all items, not seasonally adjusted, with 1982–1984=100 reference base.
```